src/test_l2_tvSettings.c

```diff
@@ -3709,6 +3709,70 @@ void test_l2_tvSettings_RetrieveLDIMShortCircuitStatus(void)
     UT_LOG_INFO("Out %s\n", __FUNCTION__);
 }
 
+  
+/**
+* @brief Test to verify the functionality of GetNumberOfDimmingZones API
+*
+* This test case verifies the functionality of the GetNumberOfDimmingZones API.
+* It checks if the API is able to correctly retrieve the  dimming zone count
+* in the TV settings. The test case invokes the TvInit() function to initialise the
+* TV settings, then it calls the GetNumberOfDimmingZones() function to get the
+* number of zones supported in current platform.
+* Finally, the test case calls the TvTerm() function to terminate the TV settings
+* and asserts that its return value is as expected.
+*
+* **Test Group ID:** 02@n
+* **Test Case ID:** 051@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation [tv-settings_L2_Low_Level_Test_Spec.md](../docs/pages/tv-settings_L2_Low_Level_Test_Spec.md)
+*/
+
+void test_l2_tvSettings_GetNumberOfDimmingZones(void)
+{
+    gTestID = 51;
+    UT_LOG_INFO("In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    tvError_t ret = tvERROR_NONE;
+    int zoneCount_actual = 0;
+    int zoneCount_expected = UT_KVP_PROFILE_GET_UINT32("tvSettings/LDIMShortCircuitStatus/zone_count");
```

Comment:
In the yaml, I can see as below

  LDIMShortCircuitStatus:
    enable: true
    size : 160

I am not seeing zone_count in the yaml. Correct me if I am wrong.

Reply:
Thanks.  Updated the zone_count in yaml.